explainaboard/metrics/nlg_meta_evaluation.py

```diff
@@ -18,6 +19,142 @@
 from explainaboard.utils.typing_utils import narrow, unwrap_or
 
 
+@dataclass
+@common_registry.register("CorrelationNLGConfig")
+class CorrelationNLGConfig(MetricConfig):
+    """Configuration of a correlation for general NLG tasks.
+
+    Args:
+        level: there are following levels: sample level, system level and dataset level
+                See more details: https://aclanthology.org/2020.emnlp-main.751.pdf
+        func_name: the method for calculating the correlation (e.g., Spearman)
+    """
+
+    level: str = "sample"
+    func_name: str = "spearmanr"
```

Comment:
where these defaults come from?

Reply:
I finally renamed `level` as `group_by` to keep consistent with how `meta_evaluation_wmt_da` task names it.